source/iterable.js

```diff
@@ -58,10 +59,12 @@ export const combineAsyncIterators = async function * (...iterators) {
 	}
 };
 
-const getNext = async iterator => {
+const getNext = async (iterator, index, {ignoredIteration}) => {
 	try {
 		return await iterator.next();
 	} catch (error) {
-		await iterator.throw(error);
+		return ignoredIteration[index] && !ignoredIteration.every(Boolean)
```

Comment:
When iterating over `subprocess[Symbol.asyncIterator]`, it is ok if either the `stdout` or `stderr` is `'inherit'`, but not if both are.